succdatroot/Makefile

```diff
@@ -4,6 +4,6 @@ include $(THEOS)/makefiles/common.mk
 
 TOOL_NAME = succdatroot
 succdatroot_FILES = main.c
-succdatroot_CODESIGN_FLAGS = -Sent.plist
+succdatroot_CODESIGN_FLAGS = -S../Entitlements.plist
```

Comment:
I noticed that `compile` script will re-sign `succdatroot` binary with the global entitlements anyway so.